### PR TITLE
remove a non-haiku

### PR DESCRIPTION
**Pull request by Haroenv**
Since the first line wasn't a haiku, (wrong amount of syllables).

### CODE REVIEW

**Comment by notwaldorf** [comment]
Caaaaaaan we make it into a haiku? 🙀

**Comment by notwaldorf** [comment]
Maybe "This is cat, hello./Where is everybody/I'm getting hungry"?

**Comment by Haroenv** [comment]
Yep, converting it to a haiku is a better option.

File: server.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ app.get('/notification-data.json', function (req, res) {
   // From http://www.adoptacatfoundation.org/cat_haikus.htm.
   // If you want to contribute with a haiku, please send a PR! :)
   var haikus = [
-      'Is anybody there?\n\nHello?',
+      'This is cat, hello.\nWhere is everybody\nI\'m getting hungry'
```
Comment (notwaldorf) [inline server.js]: Needs a comma at the end :)

**Comment by Haroenv** [comment]
Sorry, terrible at mobile code

**Comment by notwaldorf** [comment]
Heh no worries. Thanks!

**Pull request merged**